src/libraries/System.Linq.Queryable/src/System/Linq/Queryable.cs

```diff
@@ -178,6 +178,104 @@ private static Expression GetSourceExpression<TSource>(IEnumerable<TSource> sour
             return q != null ? q.Expression : Expression.Constant(source, typeof(IEnumerable<TSource>));
         }
 
+        /// <summary>
+        /// Correlates the elements of two sequences based on matching keys.
+        /// The default equality comparer is used to compare keys.
```

Comment:
Nit: try to keep summary text concise and restricted to one sentence. The part about using the default equality comparer could be moved to the remarks section.

Reply:
Yeah, the idea was to already make clear the difference between Join and LeftJoin/RightJoin right at the summary - but will remove.